Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/MenuManager.cpp

```diff
@@ -587,10 +589,30 @@ namespace AzToolsFramework
                 "Menu Manager - Could not display menu \"%s\" - menu has not been registered.", menuIdentifier.c_str()));
         }
 
+        m_lastDisplayedMenuIdentifier = menuIdentifier;
         menuIterator->second.DisplayUnderCursor();
+
         return AZ::Success();
     }
 
+    MenuManagerPositionResult MenuManager::GetLastContextMenuPosition() const
+    {
+        if (m_lastDisplayedMenuIdentifier.empty())
+        {
+            return AZ::Failure(AZStd::string::format(
+                "Menu Manager - Could not return last context menu position. No menu was displayed yet."));
```

Comment:
This doesn't look like a format string, no need for `::format`
```suggestion
            return AZ::Failure(AZStd::string(
                "Menu Manager - Could not return last context menu position. No menu was displayed yet."));
```

Reply:
I copied a different message that needed it... fixing :)